### PR TITLE
Old rustfmt version used

### PR DESCRIPTION
**Pull request by r8d8**
Updated to latest & edited travis config

### CODE REVIEW

**Comment by sjmackenzie** [comment]
All these sweeping code changes cause conflicts. Might I suggest we hold off for a while till the code settles a bit?

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ notifications:
     on_success: never
 before_install:
 #  - rustup target add $TARGET || true
-  - travis_wait cargo install rustfmt --force --vers 0.8.4 || true
+  - travis_wait cargo install rustfmt --force || true
```
Comment (sorpaas) [inline .travis.yml]: Mind to add the version? It would work better to add `--vers 0.9.0` so it won't break again in the future.

Comment (sorpaas) [inline .travis.yml]: Basically `s/cargo install rustfmt --force/cargo install rustfmt --force --vers 0.9.0`.

**Comment by sorpaas** [comment]
@sjmackenzie It's up to you and @r8d8 about when we update the `rustfmt` version, but you can do a simple `cargo fmt --all` and all your conflicts should be resolved.

**Comment by sorpaas** [comment]
@sjmackenzie I don't use nixcrates or NixOS's build system to build rustfmt.

I directly run `cargo install rustfmt --vers 0.9.0 --force` and it gives me a rustfmt at `~/.cargo/bin`. So I can then just do `PATH=~/.cargo/bin:$PATH cargo fmt --all`.

**Comment by sjmackenzie** [comment]
Ah okay I ran `$ nix-env -i rustfmt` a long time ago, this conflicted. Now
it works.

**Pull request merged**